client/settings/fraud-protection/advanced-settings/test/rule-card-notice.test.js

```diff
@@ -15,9 +15,6 @@ describe( 'Fraud protection rule card notice tests', () => {
 			</FraudProtectionRuleCardNotice>
 		);
 		expect( container ).toMatchSnapshot();
-		expect(
-			container.queryByTestId( 'rule-card-notice-info-icon-svg' )
-		).toBeInTheDocument();
```

Comment:
This and the 2 tests below this used to check that SVGs elements were included with defined test element IDs. 

Because these SVGs were removed I couldn't run these checks anymore. In any case it didn't seem super important to check that the right icon is being shown.